util/devel/lookForBadRTCalls

```diff
@@ -17,6 +17,10 @@ alloc wrapper (chpl-mem-sys.h)
 Also checks for calls to exit. We want to use chpl_exit_* instead so there's a
 common place to break.
 
+Skip checking files in 'runtime/etc/src', because right now this directory
+only contains C code for multilocale-libaries, and the shims there may call C
+standard library functions when defined for the client.
+
```

Comment:
There's no reason `runtime/etc/src` couldn't contain other code that we wouldn't want to call `exit()` or c allocation routines in the future, so I think we should make this more specific. Is there any reason your code can't include `chpl-mem-sys.h` and use those wrappers?